src/sentry/models/auditlogentry.py

```diff
@@ -14,6 +14,21 @@
 MAX_ACTOR_LABEL_LENGTH = 64
 
 
+def is_scim_token_actor(actor):
+    scim_prefix = "scim-internal-integration-"
+    scim_regex = re.compile(
+        scim_prefix
+        + r"([0-9a-fA-F]{6})\-[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{7}"
+    )
+    scim_match = re.match(scim_regex, actor.get_display_name())
```

Comment:
looking at this now i'm wondering if we even need the uuid regex here. i don't think we do. the prefix should be enough

Reply:
i'm doing this here to also get the uuid prefix all in one go. would it be better to just check if the prefix is in the display name and then get the uuid prefix later? this function is duplicate of override_actor_id

Reply:
yeah i think so.